great_expectations/compatibility/sqlalchemy.py

```diff
@@ -9,8 +9,10 @@
 
 try:
     import sqlalchemy
+    from sqlalchemy.sql.selectable import Subquery
```

Comment:
Hello, @SaeedFarahani -- thank you for this contribution.  One small change request: Please create a separate section for the new `Subquery` attribute.  Thank you.

Reply:
Hi @alexsherstinsky 
It's my pleasure, I fixed the sections

Reply:
@SaeedFarahani Much appreciated!  The code looks good -- once all the tests pass, I will approve.  Thank you very much!